### PR TITLE
feat: Support configurable commit messages in version command

### PR DESCRIPTION
**Pull request by shyndman**
The `version` command's `--message` option can be optionally supplied to set a custom commit message when updating versions. If not provided, the previous default is used.

The option's value is treated as a template with a single template var, `{new_package_versions}`, which is replaced with the list of newly versioned packages. Additionally, escaped newlines are evaluated as proper newlines, making it easier to supply a multiline string to the option.

Example
```sh
melos version --message='chore(emergency_release): Hotfix is getting deployed now!\n\n{new_package_versions}'
```

I decided to include a template library to perform the substitution, and picked one that appears to be well-built and small — `mustache_templates`. I could've written something quick and dirty instead, but it didn't feel right to have it live here.

The option is only a first step — we could go further with this, and support defaults in the `melos.yaml` file (like Lerna does with their config), but the options seems like a good first step.

Related to #69

### CODE REVIEW

File: packages/melos/lib/src/command/version.dart [hunk 1]
```diff
@@ -95,7 +115,7 @@ class VersionCommand extends MelosCommand {
       'all',
       abbr: 'a',
       negatable: false,
-      help: 'Verion private packages that are skipped by default.',
+      help: 'Version private packages that are skipped by default.',
```
Comment (Salakar) [inline packages/melos/lib/src/command/version.dart:118]: Woops 👀

File: packages/melos/lib/src/command/version.dart [hunk 2]
```diff
@@ -19,6 +19,7 @@ import 'dart:io';
 
 import 'package:ansi_styles/ansi_styles.dart';
 import 'package:conventional_commit/conventional_commit.dart';
+import 'package:mustache_template/mustache.dart';
```
Comment (Salakar) [inline packages/melos/lib/src/command/version.dart:22]: Seems like a reasonable choice for templating 👍

**Review by Salakar — APPROVED** [review]

**Comment by Salakar** [comment]
Thanks for this useful addition 🥳

**Comment by Salakar** [comment]
Tested this out with the latest release of `melos` - for some reason the line breaks didn't come through correctly, might be the wrong format for git, not sure: https://github.com/invertase/melos/commit/928deca384c4b5dc472329163fbef6738fd2426d

**Comment by shyndman** [comment]
Can you bring up the exact command you wrote? I'll get this fixed up.

**Comment by shyndman** [comment]
Nevermind, I repro'd. Give me a few.

**Comment by shyndman** [comment]
I see the issue. I'm not sure how I got this working yesterday, but I was in a bit of shell hell, trying to figure out newlines.

Fix coming right up.

**Comment by shyndman** [comment]
#73

**Pull request merged**